components/trello/trello.app.js

```diff
@@ -129,109 +74,88 @@ module.exports = {
-      };
-      return (await this._makeRequest(config)).data;
+      return await this.getResource(`boards/${id}`);
     },
     async getBoards(id) {
-      const config = {
-        url: `${await this._getBaseUrl()}/members/${id}/boards`,
-      };
-      return (await this._makeRequest(config)).data;
+      return await this.getResource(`members/${id}/boards`);
     },
     async getCard(id) {
-      const config = {
-        url: `${await this._getBaseUrl()}/cards/${id}`,
-      };
-      return (await this._makeRequest(config)).data;
+      return await this.getResource(`cards/${id}`);
     },
     async getCards(id) {
-      const config = {
-        url: `${await this._getBaseUrl()}/boards/${id}/cards`,
-      };
-      return (await this._makeRequest(config)).data;
+      return await this.getResource(`boards/${id}/cards`);
     },
     async getChecklist(id) {
-      const config = {
-        url: `${await this._getBaseUrl()}/checklists/${id}`,
-      };
-      return (await this._makeRequest(config)).data;
+      return await this.getResource(`checklists/${id}`);
     },
     async getLabel(id) {
-      const config = {
-        url: `${await this._getBaseUrl()}/labels/${id}`,
-      };
-      return (await this._makeRequest(config)).data;
+      return await this.getResource(`labels/${id}`);
     },
     async getList(id) {
-      const config = {
-        url: `${await this._getBaseUrl()}/lists/${id}`,
-      };
-      return (await this._makeRequest(config)).data;
+      return await this.getResource(`lists/${id}`);
     },
     async getLists(id) {
-      const config = {
-        url: `${await this._getBaseUrl()}/boards/${id}/lists`,
-      };
-      return (await this._makeRequest(config)).data;
+      return await this.getResource(`boards/${id}/lists`);
     },
     async getNotifications(id, params) {
-      const config = {
-        url: `${await this._getBaseUrl()}/members/${id}/notifications`,
-        params,
-      };
-      return (await this._makeRequest(config)).data;
+      return await this.getResource(`members/${id}/notifications`, params);
     },
     async getMember(id) {
-      const config = {
-        url: `${await this._getBaseUrl()}/members/${id}`,
-      };
-      return (await this._makeRequest(config)).data;
+      return await this.getResource(`members/${id}`);
+    },
+    async getAttachment(cardId, attachmentId) {
+      return await this.getResource(
+        `cards/${cardId}/attachments/${attachmentId}`
+      );
     },
     async createHook({ id, endpoint }) {
-      console.log(`id: ${id}`);
       const resp = await this._makeRequest({
         method: "post",
-        url: `${await this._getBaseUrl()}/webhooks/`,
+        url: `${this._getBaseUrl()}webhooks/`,
         headers: {
           "Content-Type": "applicaton/json",
         },
         params: {
           idModel: id,
-          description: "Pipedream Source ID", //todo add ID
+          description: "Pipedream Source ID",
```

Comment:
**-Minor-**
I think the `TODO` here is to add the event source's ID to the description.
We should keep the comment since it's still not done.